Description: Fix test error with insalled rebar and proper package on Debian
Forwarded: not-needed
Bug-Debian: None
Author: Nobuhiro Iwamatsu <iwamatsu@debian.org>

From 48d598c0e7f45d2ba17fd4fd173877ce6372a590 Mon Sep 17 00:00:00 2001
From: Nobuhiro Iwamatsu <iwamatsu@nigauri.org>
Date: Wed, 28 Nov 2012 09:12:00 +0900
Subject: [PATCH 2/2] Remove deps/proper from tests and inittests target

Signed-off-by: Nobuhiro Iwamatsu <iwamatsu@nigauri.org>
---
 Makefile |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 736d794..8bbf934 100644
--- a/Makefile
+++ b/Makefile
@@ -35,9 +35,9 @@ deps/proper:
 	@$(REBAR) -C rebar.tests.config get-deps
 	cd deps/proper && $(REBAR) compile
 
-tests: clean deps/proper app eunit ct
+tests: clean app eunit ct
 
-inttests: clean deps/proper app eunit intct
+inttests: clean app eunit intct
 
 eunit:
 	@$(REBAR) -C rebar.tests.config eunit skip_deps=true
-- 
1.7.10.4

